src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -896,6 +900,10 @@ FixedwingPositionControl::control_auto_descend(const hrt_abstime &now)
 uint8_t
 FixedwingPositionControl::handle_setpoint_type(const uint8_t setpoint_type, const position_setpoint_s &pos_sp_curr)
 {
+	if (!_control_mode.flag_control_position_enabled && _control_mode.flag_control_velocity_enabled) {
+		return position_setpoint_s::SETPOINT_TYPE_VELOCITY;
```

Comment:
@dagar We are not using `SETPOINT_TYPE_VELOCITY` from the navigator, but we still need `SETPOINT_TYPE_VELOCITY` in the fw position controller

Reply:
If it's easy we can keep it for now, but we'll have to do something about it later.

Reply:
@dagar I think this needs to be changed all together on moving away from position setpoint as a state machine for the position controller behavior